legacy/src/api/integrations/managed-integrations.md

```diff
@@ -133,22 +116,12 @@ A summary of the [Invitation][] for a Managed Integration.
 {% json %}
 {
   "id": "5ee0c486308f590260d9a07f",
-  "type": "kete-enrolment",
+  "type": "paypal-referral",
   "externalId": "DKTs3U38hdhfEqwF1JKoT2",
   "enabled": true,
   "params": {
-    "bankAccounts": [
-      { "number": "00-1111-2222222-00" },
-    ],
-    "givenName": "John",
-    "familyName": "Doe",
-    "email": "john@doe.com",
-    "phoneNumber": "+64270000000"
-  },
-  "invitation": {
```

Comment:
A `paypal-referral` managed integration type does not have an invitation attached.